pages/index.html

```diff
@@ -44,107 +44,93 @@
-                                    <span class="CoveoFieldValue" data-field="@language" data-text-caption="Language" style="margin-right:30px;"></span>
-                                    <span class="CoveoFieldValue" data-field="@filetype" data-text-caption="File Type" style="margin-right:30px;"></span>
-                                  </div>
-                                </div>
-                                <div class="coveo-result-row" style="margin-top:10px;">
-                                  <div class="coveo-result-cell">
-                                    <div class="CoveoPrintableUri"></div>
-                                  </div>
+                              <div class="coveo-result-cell" style="width:120px;text-align:right;font-size:12px">
+                                <div class="coveo-result-row">
+                                  <span class="CoveoFieldValue" data-field="@date" data-helper="date"></span>
                                 </div>
                               </div>
                             </div>
-                        </script>
-                    </div>
-                    <div class="CoveoResultList" data-layout="card" data-wait-animation="fade" data-auto-select-fields-to-include="true"></div>
-                    <div class="CoveoPager"></div>
-                    <div class="CoveoLogo"></div>
-                    <div class="CoveoResultsPerPage"></div>
+                            <div class="coveo-result-row" style="margin-top:10px;">
+                              <div class="coveo-result-cell">
+                                <span class="CoveoExcerpt"></span>
+                              </div>
+                            </div>
+                            <div class="coveo-result-row" style="margin-top:10px;">
+                              <div class="coveo-result-cell">
+                                <span class="CoveoFieldValue" data-field="@author" data-text-caption="Author" style="margin-right:30px;"></span>
+                                <span class="CoveoFieldValue" data-field="@source" data-text-caption="Source" style="margin-right:30px;"></span>
+                                <span class="CoveoFieldValue" data-field="@language" data-text-caption="Language" style="margin-right:30px;"></span>
+                                <span class="CoveoFieldValue" data-field="@filetype" data-text-caption="File Type" style="margin-right:30px;"></span>
+                              </div>
+                            </div>
+                            <div class="coveo-result-row" style="margin-top:10px;">
+                              <div class="coveo-result-cell">
+                                <div class="CoveoPrintableUri"></div>
+                              </div>
+                            </div>
+                            <div class="coveo-result-row" style="margin-top:10px;">
```

Comment:
So by default we put it in a row below each result?

Reply:
Btw, just to clarify: `pages/index.html` is used for development only.

Reply:
It can still be a model for what kind of page will need to be created no?

Reply:
In the jsui, the folder `pages/` contains a lot of example pages that people sometimes copy and paste.